src/SourceGenerators/Uno.UI.SourceGenerators/TSBindings/TSBindingsGenerator.cs

```diff
@@ -53,17 +39,8 @@ public void Execute(GeneratorExecutionContext context)
 
 				if (!string.IsNullOrEmpty(_bindingsPaths))
 				{
-					_stringSymbol = context.Compilation.GetTypeByMetadataName("System.String");
-					_intSymbol = context.Compilation.GetTypeByMetadataName("System.Int32");
-					_uintSymbol = context.Compilation.GetTypeByMetadataName("System.UInt32");
-					_floatSymbol = context.Compilation.GetTypeByMetadataName("System.Single");
-					_doubleSymbol = context.Compilation.GetTypeByMetadataName("System.Double");
-					_byteSymbol = context.Compilation.GetTypeByMetadataName("System.Byte");
-					_shortSymbol = context.Compilation.GetTypeByMetadataName("System.Int16");
```

Comment:
Removing these may make for better performance since `GetTypeByMetadataName` isn't super fast compared to jsut checking SpecialType